templates/partials/answers/address.html

```diff
@@ -0,0 +1,35 @@
+{% from "components/address/_macro.njk" import onsAddressInput %}
+
+{% set input = form.fields[answer.id] if input is undefined else input %}
```

Comment:
It would be clearer here if `input` was named `address_form`. When is `input` not defined?

Reply:
Renamed + removed the if as it is always defined.